processCluster.ts

```diff
@@ -106,78 +65,68 @@ export const processCluster = async (
       };
       // get distance between pixel and landing site
       let centerOfBiomassDistanceToLanding = response.waypoints[0].distance;
-      centerOfBiomassDistanceToLanding = centerOfBiomassDistanceToLanding * metersToFeetConstant; // feet
+      centerOfBiomassDistanceToLanding = centerOfBiomassDistanceToLanding *metersToFeetConstant ; // meters to feet conversion
+      const mean_yarding = ((0.002*(centerOfBiomassDistanceToLanding**2)+0.044*(centerOfBiomassDistanceToLanding))+137.5815) * metersToFeetConstant; // feet
 
       // get landing elevation
       const landingElevationInMeters = await getElevation(landing.latitude, landing.longitude);
       const landingElevation = landingElevationInMeters * metersToFeetConstant;
 
-      const area = pixels.length * pixelsToAcreConstant; // pixels are 30m^2, area needs to be in acres
+      const area = 32.024857 ; // pixels are 1.296000e+05m^2 according to the expanse() function in R, area needs to be in acres
 
       const centerOfBiomassElevationInMeters = await getElevation(
         centerOfBiomassLat,
         centerOfBiomassLng
       );
       const centerOfBiomassElevation = centerOfBiomassElevationInMeters * metersToFeetConstant;
+      
+      let land_own = "private"
+      if (firstCluster.Land_Ownership !== null && firstCluster.Land_Ownership !== undefined && firstCluster.Land_Ownership !== '') {
```

Comment:
all of this stuff is the same as `if (firstCluster.Land_Ownership) { ... }`.  That will make sure the variable isn't null, undefined or empty.